### PR TITLE
Enhance the plot for the channels of infections.

### PR DESCRIPTION
**Pull request by tobiasraabe**
### Current behavior

What is the current state?

### Desired behavior

What do you want to achieve?

### Solution / Implementation

What is your solution?

### Todo

- [ ] Review whether the documentation needs to be updated.
- [ ] Document PR in CHANGES.rst.

### CODE REVIEW

File: src/sid/plotting.py [hunk 1]
```diff
@@ -171,11 +176,54 @@ def _create_y_ticks_and_labels(df):
...
+    "cmap": "YlOrBr",
 }
 
 
-def plot_infection_rates_by_contact_models(df_or_time_series, fig_kwargs=None):
-    """Plot infection rates by contact models."""
+def plot_infection_rates_by_contact_models(
+    df_or_time_series: Union[pd.DataFrame, dd.core.DataFrame],
+    show_reported_cases: bool = False,
+    numbers: str = "shares",
```
Comment (janosg) [inline src/sid/plotting.py:186]: I think `numbers` is not a very intuitive name. Maybe `units` or `metric` instead?

**Review by janosg — APPROVED** [review]
Nice!

File: src/sid/plotting.py (hunk 1 above)
Comment (tobiasraabe) [reply src/sid/plotting.py:186]: agree. then ``unit`` and singular for the values like ``share``, ``population_share``, ``incidence``?

Comment (janosg) [reply src/sid/plotting.py:186]: Yes!

**Pull request merged**